docs/content/latest/api/ysql/datatypes/type_array/functions-operators/any-all.md

```diff
@@ -0,0 +1,306 @@
+---
+title: ANY and ALL—test if an element is in an array
+linkTitle: =ANY and =ALL
+headerTitle: =ANY and =ALL — test if an element is in an array
+description: The =ANY and =ALL operators test if an element is in an array
+menu:
+  latest:
+    identifier: any-all
+    parent: array-functions-operators
+    weight: 10
+isTocNested: true
+showAsideToc: true
+---
+
+## Overview
+
+**Purpose:** Return `TRUE`, `FALSE`, or `NULL` according to the outcome of the particular comparison, `=ANY` or `=ALL`, between the input [LHS](https://en.wikipedia.org/wiki/Sides_of_an_equation) _element_ and input [RHS](https://en.wikipedia.org/wiki/Sides_of_an_equation) array. Each of the LHS and the RHS is, in general, an expression. Further, the RHS expression must be parenthesized.
+
+The value to which the LHS evaluates is compared, using the appropriate data type overload of the `=` operator, to each of the RHS expression's evaluated _elements_ to yield a `boolean` result (`TRUE`, `FALSE`, or `NULL`) according to the normal rules for _element_ comparison. (See the [Semantics demonstration](./#semantics-demonstration) section.)
+
+**Signature**
+
+Each of the `=ANY` and `=ALL` operators has the same signature, thus:
+
+```
+input value:       anyelement, anyarray
+return value:      boolean
+```
+**Note:** The term of art _element_ is used in this section for what the pseudo-type name `anyelement` represents. Each of these operators requires that the data type of the _element_ value to which the LHS expression evaluates corresponds to the data type of the array value to which the RHS expression evaluates. For example, when the data type of the LHS _element_ is, say _"t"_, then the RHS array's data type must be _"t[]"_. The RHS array can have any dimensionality. The operators are sensitive only to the actual _elements_ in the array.
+
+Think of `=ANY` and `=ALL` as the spellings of single [binary operators](https://en.wikipedia.org/wiki/Binary_operation). Notice that, `<> ANY`, `<> ALL`, `< ANY`, `>= ALL`, and so on are syntactically legal, and produce outcomes without error. But these outcomes are semantically meaningless.
+
+`=ANY` is functionally equivalent to `IN` (but `IN` is illegal syntax when the RHS is an array).
+
+`=ALL` has no functional equivalent in the way that `=ANY` is functionally equivalent to `IN`. Briefly (and leaving the possibility of `NULL` to the [semantics of the `=ALL` operator](./#the-semantics-of-the-all-operator) section, `=ALL` produces `TRUE` when every _element_ in the RHS array is equal to every other of its _elements_ and when the LHS _element_ is also equal to that value. 
+
+## The semantics of the =ANY operator
```

Comment:
Does this generate correctly? Assuming so, but any special characters in headers can cause unexpected results in Hugo at times.

Reply:
Yes, they generate properly. See, for example, [HERE](http://localhost:1313/latest/api/ysql/datatypes/type_array/functions-operators/comparison/#examples-for-each-operator). It's been out there in the Internet for ages. It looks fine. Marking as "resolved".